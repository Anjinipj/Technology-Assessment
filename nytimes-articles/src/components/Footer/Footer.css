footer {
  display: flex;
  justify-content: space-between;
  margin-right: 2rem;
  margin-left: 2rem;
}

footer h3 {
  font-weight: normal;
  color: rgb(87, 85, 85);
  font-size: 1em;
  font-style: italic;
}

footer a:hover {
  color: black;
}

.footer-container {
  flex: none;
  width: 100%;
  height: 7rem;
  border-top: rgb(219, 218, 218) solid 1px;
}

@media (max-width: 1000px) {
  footer {
    flex-direction: column;
    align-items: center;
  }
}
