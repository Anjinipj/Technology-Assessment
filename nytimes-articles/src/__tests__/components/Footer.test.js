import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../../components/Footer/Footer';

it('renders correctly Header ', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});