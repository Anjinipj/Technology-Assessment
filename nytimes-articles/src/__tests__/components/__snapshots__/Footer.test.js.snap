// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Header  1`] = `
<div
  className="footer-container"
>
  <footer>
    <h3>
      Â© 2024 The New York Times Company
    </h3>
    <div
      className="personal-container"
    >
      <h3
        className="personal-info"
      >
        Web Development by Anjini Jayachandran
      </h3>
    </div>
  </footer>
</div>
`;
